experimental/FTheoryTools/test/tate_models.jl

```diff
@@ -233,7 +233,7 @@ end
   irr_i5_s = irrelevant_ideal(tas);
   sri_i5_s = stanley_reisner_ideal(tas);
   lin_i5_s = ideal_of_linear_relations(tas);
-  id_fin,  = _blowup_global_sequence(id_i5_s, [[7, 8, 6], [2, 3, 1], [3, 4], [2, 4]], irr_i5_s, sri_i5_s, lin_i5_s)
+  id_fin,  = _blowup_global_sequence(id_i5_s, [[8, 9, 6], [2, 3, 1], [3, 4], [2, 4]], irr_i5_s, sri_i5_s, lin_i5_s)
```

Comment:
Why is this change needed? I was under the impression that this change would only affect the behavior "under the hood", but this change indicates differently. Or was there a mistake in the tests to begin with?

Reply:
This change is needed because an additional coordinate was added to the base space for the tests, in order to make the gauge locus more complicated